src/util/assertString.js

```diff
@@ -0,0 +1,11 @@
+export default function assertString(input) {
+  if (typeof input !== 'string') {
+    if (process.env.NODE_ENV === 'node') {
```

Comment:
We'll be dropping the deprecation notice and dropping the depd dependency for v5.